C-git-commands.asc

```diff
@@ -59,7 +59,7 @@ Accompanying the configuration instructions in <<ch01-getting-started#_editor>>,
 |Kate (Linux) |`git config --global core.editor "kate --block"`
 |nano |`git config --global core.editor "nano -w"`
 |Notepad (Windows 64-bit) |`git config core.editor notepad`
-|Notepad++ (Windows 64-bit) |`git config --global core.editor "'C:\Program Files\Notepad++\notepad++.exe' -multiInst -notabbar -nosession -noPlugin"` (Also see note below)
+|Notepad++ (Windows 64-bit) |`git config --global core.editor "'C:\Program Files\Notepad+\+\notepad++.exe' -multiInst -notabbar -nosession -noPlugin"` (Also see note below)
```

Comment:
Interesting. Only the second `+` needs to be escaped? The other three are fine?

Reply:
I believe escaping any one of the four `+` that surrounds `\notepad` should stop it from being considered as an enclosing pair of double pluses.

When I experimented with escaping more than one, the literal backslashes appeared in the output, possibly because by that point there was no special meaning to actually escape and they were interpreted literally.